openslides/core/static/js/core/projector.js

```diff
@@ -171,7 +171,7 @@ angular.module('OpenSlidesApp.core.projector', ['OpenSlidesApp.core'])
         var STEPS = 5;
         $scope.scroll = 0;
         var setScroll = function (scroll) {
-            scroll = 80 * scroll;
+            scroll = -80 * scroll;
```

Comment:
Right, currently we use "natural scrolling". But this is not very intuitive with our arrow buttons. So,  it's very good that you fix this here!